src/components/Book.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ id, title, category }) => (
+  <tr>
+    <td>{id}</td>
+    <td>{title}</td>
+    <td>{category}</td>
+  </tr>
```

Comment:
add `Remove book` button to the book 
(
it attaches passed `handleRemoveBook` to on click event of `Remove book` button)